internal/course/database/course_queries.go

```diff
@@ -1,27 +1,39 @@
 package database
 
+import (
+	"fmt"
+)
+
 const (
-	createCourse       = "create course"
-	deleteCourse       = "delete course by uuid"
-	getCourse          = "get course by uuid"
-	listCourse         = "list course"
-	updateCourseByID   = "update course by uuid"
-	updateCourseByCode = "update course by code"
+	returning_columns = "uuid, code, name, underline, image, image_cover, excerpt, description, created_at, updated_at"
+	// CREATE.
+	createCourse = "create course"
+	// READ.
+	listCourse = "list course"
+	getCourse  = "get course by uuid"
+	// UPDATE.
+	updateCourseByUUID = "update course by uuid"
```

Comment:
Well, if `getCourse` also gets it by uuid and doesn't have it explicitly on its name, I would probably rename it to `updateCourse`.

Reply:
https://github.com/sumelms/microservice-course/pull/19/commits/6244cbf2156cfdfa417826cf5017f6c73799dd89